### ISSUE TITLE
Once we start server work - Use lat/long data to call weather API

### ISSUE THREAD

**Issue by adk1010** [tomrbland/webTech#21]

**Comment by tomrbland** [comment]
Fyi there are Global variables for lat and long

**Comment by adk1010** [comment]
Ok, hopefully that means that it won't be too tricky

**Comment by tomrbland** [comment]
Yeah it'll be fine. They're in maps.js so as long as the weather,js in included after maps.js then it should work no problem. You might want to merge master in to weather to bring it up to date though. I noticed that it's behind a bit on the network diagram.

**Comment by adk1010** [comment]
Yeah, will do
